test/bats.bats

```diff
@@ -3,10 +3,18 @@
 load test_helper
 fixtures bats
 
-@test "no arguments prints usage instructions" {
+@test "no arguments prints message and usage instructions" {
   run bats
   [ $status -eq 1 ]
-  [ $(expr "${lines[1]}" : "Usage:") -ne 0 ]
+  [ $(expr "${lines[1]}" : "Error: Must specify at least one") -ne 0 ]
```

Comment:
Extra credit: Replace the `"$(expr ...) -ne 0` expressions with `=` expressions like `${lines[1]} = "Error: Must specify at lest one "*`. Helps the test run a little faster by avoiding subshells/subprocesses.

Reply:
will take a look ...

Reply:
Actually, I'm a little off base here; `expr` is how we're able to do regular expressions using `[  ]`, since we can't rely on `[[ ]]` working under 3.2.57(1)-release on macOS. So nevermind this comment.